dstk-infra/bin/storage

```diff
@@ -69,7 +69,7 @@ storage__readiness() {
 storage__upgrade() {
   # Enumerate patches
   SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd -P)
-  readarray -d '' patches < <(find "${SCRIPT_DIR}/../postgres/patches" -type f -name '*.sql' -print0 | sort)
+  readarray -d '' patches < <(find "${SCRIPT_DIR}/../postgres/patches" -type f -name '*.sql' -print0 | sort -z)
```

Comment:
for clarification on this change, `sort` normally expects records to be newline-terminated. In this case, however, the `-print0` flag separates records with a `NUL` character. Using `sort -z` changes the behavior to look for the `NUL` delimiter so that we're able to properly sort the patches by date